<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dashboard_widget</class>
 <widget class="QWidget" name="Dashboard_widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QFrame" name="frame_dashboard">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>480</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>800</width>
     <height>480</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>800</width>
     <height>480</height>
    </size>
   </property>
   <widget class="QFrame" name="frame_gauge">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>35</y>
      <width>400</width>
      <height>360</height>
     </rect>
    </property>
    <widget class="QFrame" name="gaugeBackground">
     <property name="geometry">
      <rect>
       <x>64</x>
       <y>60</y>
       <width>270</width>
       <height>270</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">
QFrame{
background-color: rgb(20, 20, 20);
border-radius:135px;
}</string>
     </property>
     <widget class="QFrame" name="gaugeInnerCircle">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>50</y>
        <width>170</width>
        <height>170</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QFrame{
	background-color: rgb(0, 0, 0);
	
	border-radius:85px;
}



</string>
      </property>
      <widget class="QFrame" name="frame_km">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>150</width>
         <height>150</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>150</width>
         <height>150</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>8</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QFrame{
	background:none; 
	border-radius:75px;
	
	border:2px solid rgb(255,0,0);
}
QLabel{
	border:none;
	color:white;
}</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>10</number>
        </property>
        <property name="topMargin">
         <number>10</number>
        </property>
        <property name="rightMargin">
         <number>10</number>
        </property>
        <property name="bottomMargin">
         <number>10</number>
        </property>
        <item>
         <widget class="QLabel" name="label_kmh">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>38</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>38</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Eras Light ITC</family>
            <pointsize>18</pointsize>
            <italic>true</italic>
           </font>
          </property>
          <property name="text">
           <string>KM/H </string>
          </property>
          <property name="alignment">
           <set>Qt::AlignBottom|Qt::AlignHCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_vel">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>70</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>9999</width>
            <height>9999</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>DS-Digital</family>
            <pointsize>58</pointsize>
            <weight>50</weight>
            <italic>false</italic>
            <bold>false</bold>
            <underline>false</underline>
            <strikeout>false</strikeout>
            <stylestrategy>PreferDefault</stylestrategy>
           </font>
          </property>
          <property name="text">
           <string>200</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignHCenter|Qt::AlignTop</set>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QFrame" name="needle">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>250</width>
        <height>250</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QFrame{
	border:none;
	background-color: qconicalgradient(cx:0.5, cy:0.5, angle:240,stop:1.0 rgba(0,0,0,0),  stop:1.0 rgba(255, 170, 0, 255), stop:1.0 rgba(0, 0, 0, 0));

	border-radius:125px;
}
</string>
      </property>
     </widget>
     <zorder>needle</zorder>
     <zorder>gaugeInnerCircle</zorder>
    </widget>
    <widget class="QFrame" name="frame_nums">
     <property name="geometry">
      <rect>
       <x>8</x>
       <y>8</y>
       <width>384</width>
       <height>356</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="vertical_layout_4">
      <item>
       <widget class="QLabel" name="label_nums">
        <property name="pixmap">
         <pixmap resource="../../resources.qrc">:/styles/Resources/speedNumbers.png</pixmap>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <zorder>frame_nums</zorder>
    <zorder>gaugeBackground</zorder>
   </widget>
   <widget class="QFrame" name="frame_player">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>80</y>
      <width>280</width>
      <height>280</height>
     </rect>
    </property>
    <widget class="QLabel" name="label_player_logo">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>0</y>
       <width>140</width>
       <height>140</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>140</width>
       <height>140</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>140</width>
       <height>140</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="../../resources.qrc">:/styles/Resources/spotify-logo.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
     <property name="margin">
      <number>0</number>
     </property>
    </widget>
    <widget class="QFrame" name="frame_media_data">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>140</y>
       <width>280</width>
       <height>65</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color:white;</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="label_artista3">
        <property name="font">
         <font>
          <family>Bahnschrift SemiLight</family>
          <pointsize>14</pointsize>
          <kerning>true</kerning>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(247, 247, 247);</string>
        </property>
        <property name="text">
         <string>El ultimo dia del resto de mi vida</string>
        </property>
        <property name="scaledContents">
         <bool>false</bool>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_artista">
        <property name="font">
         <font>
          <family>Bahnschrift SemiLight</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(186, 186, 186);</string>
        </property>
        <property name="text">
         <string>Avicci, Nicky Romero</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignHCenter|Qt::AlignTop</set>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QSlider" name="slider_duration">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>210</y>
       <width>280</width>
       <height>22</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>30</height>
      </size>
     </property>
     <property name="cursor">
      <cursorShape>SizeHorCursor</cursorShape>
     </property>
     <property name="tabletTracking">
      <bool>true</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">
QSlider::groove:horizontal{
	height:12px;	
	width:260;
	background:transparent;
}

QSlider::handle:horizontal{
	background:red;
	width:12px;
	border-radius:6px;	

}</string>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QFrame" name="frame_durationLabels">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>229</y>
       <width>280</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(94, 94, 94);</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>4</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="label_currentTime">
        <property name="font">
         <font>
          <family>Bahnschrift SemiLight</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>0:00</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_duration">
        <property name="font">
         <font>
          <family>Bahnschrift SemiLight</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>3:23</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="Line" name="line">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>221</y>
       <width>260</width>
       <height>1</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border:1px solid gray;</string>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <zorder>label_player_logo</zorder>
    <zorder>frame_media_data</zorder>
    <zorder>frame_durationLabels</zorder>
    <zorder>line</zorder>
    <zorder>slider_duration</zorder>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="../../resources.qrc"/>
  <include location="../../resources.qrc"/>
 </resources>
 <connections/>
</ui>
